envrc

```diff
@@ -0,0 +1,19 @@
+export deployed=false
+
+export AllowanceTarget_ADDRESS="0x62b546741A5D59733E1D7ee2f12dF3b790Fc1bb8"
```

Comment:
nit: Same as the env comment above. Use unified naming rule would improve the readability and decrease the mental overhead

Reply:
Sure, didn't notice at the time. Fixed in e8aa620 & 2b1f74d.